CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## v0.6.0
+
+* Add auto-updater (#515)
```

Comment:
This is only added in for MacOS at the moment, as the auto-updating required signed Windows images, and we weren't sure if we would have them ready or not.

Reply:
Good catch. Fixed in my local copy and will push a change once I get your confirmation that all feedback is in.